addon/components/cf-field/input/table.js

```diff
@@ -21,6 +21,10 @@ export default Component.extend({
   documentToEdit: null,
   documentToDelete: null,
 
+  parseDocument(raw) {
+    return parseDocument(raw);
+  },
```

Comment:
Why the indirection?

Reply:
Because we need to override it in the default table component

Reply:
Ah of course, thanks!